libkbfs/quota_usage.go

```diff
@@ -118,7 +118,7 @@ func (q *EventuallyConsistentQuotaUsage) Get(
 	}()
 	past := q.config.Clock().Now().Sub(c.timestamp)
 	switch {
-	case past > blockTolerance:
+	case past > blockTolerance || c.timestamp.IsZero():
```

Comment:
No, note the `math.MaxInt64` [here](https://github.com/keybase/kbfs/blob/7331354d03eb7b5d59cdc88610190974908958bc/libkbfs/backpressure_disk_limiter.go#L719).